решение.cpp

```diff
@@ -0,0 +1,48 @@
+/*#include <iostream>
+
+using namespace std;
+
+int main() {
+
+	int ten, two;
+
+	cin >> ten;
+	do {
+		two = ten % 2;
+		ten = ten / 2;
+		cout << two;
+	} while (ten != 0);
+	return 0;
+} */
+
+#include <iostream>
+#include <cmath>
+#include <clocale>
+#include <vector>
+
+using namespace std;
+
+int main() {
+	setlocale(LC_ALL, "rus");
+	double x;
+	int k;
+	
+	cout << "������� x (-1; 1), k (> 1): ";
+	cin >> x;
+	cin >> k;
+	cout.precision(10);
+	cout << "��������� ����� �������: " << 1 / pow((1 + x), 3) << endl;
+	int a = 2, b = 3;
+	double result = 1.0;
+	while (true) {
+		if (fabs(pow(-1, a - 1) * a * b * pow(x, (a - 1)) / 2) > pow(10, (double)-k)) {

```

Comment:
What about storing current row member in some variable

Reply:
i've accidently clicked "resolve conversation", now i'm goin to change something you has said